Resources/Prototypes/_Sunrise/Loadouts/role_loadouts.yml

```
@@ -449,3 +449,45 @@
+  nameDataset: NamesBorg
+  canCustomizeName: true
+
+- type: roleLoadout
+  id: JobMedicalBorg
+  nameDataset: NamesBorg
+  canCustomizeName: true
+
+- type: roleLoadout
+  id: JobSecurityBorg
+  nameDataset: NamesBorg
+  canCustomizeName: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)